docs/source/edb.rst

```diff
@@ -0,0 +1,15 @@
+*****
+tests
```

Comment:
I think this should say `edb` instead of `tests`?

Reply:
Whoops!  Good catch!